snowflake-table-grant-all/main.tf.json

```diff
@@ -46,6 +46,7 @@
     "privileges": [
```

Comment:
⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>local.privileges is declared but not used